src/Tracing/SpanContext.php

```diff
@@ -191,9 +194,12 @@ public function setEndTimestamp(?float $endTimestamp): void
      * @param string $header The sentry-trace header from the request
      *
      * @return static
+     *
+     * @deprecated since version 3.1, to be removed in 4.0 use {@link TransactionContext::fromSentryTrace()}
      */
     public static function fromTraceparent(string $header)
     {
+        @trigger_error(sprintf('The %s() method is deprecated since version 3.1 and will be removed in 4.0. Use TransactionContext::fromSentryTrace() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Please add a blank line below to improve readability. Some day you will get this right 😄